osh/worker/osh-worker.service

```diff
@@ -5,9 +5,10 @@ After=network-online.target
 
 [Service]
 Type=exec
+ExecReload=kill -HUP $MAINPID
 ExecStartPre=test -r /etc/osh/worker.conf
 ExecStart=/usr/sbin/osh-worker -f
-KillMode=process
+KillMode=mixed
```

Comment:
I find this part of the commit message of c98d498bb8e7ae47481f38d4387a8e1ec761d126 misleading:
```
With the current value of `KillMode=`, ...
```

I guess `current value` refers to the value before the change.  So I would either say `previous value` or expand the value directly:
```
With `KillMode=process` ...
```

Reply:
Addressed in 6731c38.